CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Features
 
+- The `pybamm_install_odes` command now includes support for macOS systems and can be used to set up SUNDIALS and install the `scikits.odes` solver on macOS ([#3417](https://github.com/pybamm-team/PyBaMM/pull/3417), [#3706](https://github.com/pybamm-team/PyBaMM/3706]))
```

Comment:
```suggestion
```

This entry can be removed, since the `scikits.odes` solver was removed in #3932.